features/steps/steps.py

```diff
@@ -0,0 +1,99 @@
+# pylint: disable=function-redefined, missing-function-docstring
+# flake8: noqa
+import logging
+import requests
+from datetime import date
+from behave import given, when, then
+from compare import expect, ensure
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import Select, WebDriverWait
+from selenium.webdriver.support import expected_conditions
+
+ID_PREFIX = "order_"
+
+@given('the following orders')
+def step_impl(context):
+    """ Delete all Orders and load new ones """
+    # List all of the orders and delete them one by one
+    rest_endpoint = f"{context.BASE_URL}/orders"
+    context.resp = requests.get(rest_endpoint)
+    expect(context.resp.status_code).to_equal(200)
+    for order in context.resp.json():
+        context.resp = requests.delete(f"{rest_endpoint}/{order['id']}")
+        expect(context.resp.status_code).to_equal(204)
+
+    # load the database with new pets
+    for row in context.table:
+        payload = {
+            "customer_id": row['customer_id'],
+            "status": row['status'],
+        }
+        context.resp = requests.post(rest_endpoint, json=payload)
+        expect(context.resp.status_code).to_equal(201)
+
+@when('I visit the "home page"')
+def step_impl(context):
+    """ Make a call to the base URL """
+    context.driver.get(context.BASE_URL)
+
+@then('I should see "{message}" in the title')
+def step_impl(context, message):
+    """ Check the document title for a message """
+    expect(context.driver.title).to_contain(message)
+
+@then('I should not see "{message}"')
+def step_impl(context, message):
+    assert message not in str(context.resp.text)
```

Comment:
Can you replace this with the following code from professor's example?

```
@then('I should not see "{text_string}"')
def step_impl(context, text_string):
    element = context.driver.find_element(By.TAG_NAME, 'body')
    error_msg = "I should not see '%s' in '%s'" % (text_string, element.text)
    ensure(text_string in element.text, False, error_msg)
```

Reply:
Will do.